<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvancedOptionsPage</class>
 <widget class="QWidget" name="AdvancedOptionsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>435</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Advanced options</string>
     </property>
     <layout class="QGridLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="5" column="0">
       <widget class="QCheckBox" name="recursively_add_files">
        <property name="text">
         <string>Include sub-folders when adding files from folder</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" rowspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QLabel" name="label_track_duration_diff">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Ignore track duration difference under this number of seconds</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="ignore_track_duration_difference_under">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::UpDownArrows</enum>
          </property>
          <property name="accelerated">
           <bool>true</bool>
          </property>
          <property name="suffix">
           <string/>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>7200</number>
          </property>
          <property name="value">
           <number>2</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_ignore_regex">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Ignore file paths matching the following regular expression:</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="regex_error">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="ignore_hidden_files">
        <property name="text">
         <string>Ignore hidden files</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="ignore_regex"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_completeness">
     <property name="title">
      <string>Ignore the following tracks when determining whether a release is complete</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="completeness_ignore_videos">
        <property name="text">
         <string>Video tracks</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="completeness_ignore_pregap">
        <property name="text">
         <string>Pregap tracks</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="completeness_ignore_data">
        <property name="text">
         <string>Data tracks</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="completeness_ignore_silence">
        <property name="text">
         <string>Silent tracks</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>181</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ignore_regex</tabstop>
  <tabstop>ignore_hidden_files</tabstop>
  <tabstop>recursively_add_files</tabstop>
  <tabstop>ignore_track_duration_difference_under</tabstop>
  <tabstop>completeness_ignore_videos</tabstop>
  <tabstop>completeness_ignore_pregap</tabstop>
  <tabstop>completeness_ignore_data</tabstop>
  <tabstop>completeness_ignore_silence</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
